langchain4j-open-ai/src/test/java/dev/langchain4j/model/openai/common/OpenAiResponsesStreamingChatModelListenerIT.java

```diff
@@ -0,0 +1,52 @@
+package dev.langchain4j.model.openai.common;
+
+import static dev.langchain4j.model.openai.OpenAiChatModelName.GPT_4_1_NANO;
+import static java.util.Collections.singletonList;
+
+import dev.langchain4j.exception.HttpException;
+import dev.langchain4j.model.chat.StreamingChatModel;
+import dev.langchain4j.model.chat.common.AbstractStreamingChatModelListenerIT;
+import dev.langchain4j.model.chat.listener.ChatModelListener;
+import dev.langchain4j.model.openai.OpenAiResponsesStreamingChatModel;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+@EnabledIfEnvironmentVariable(named = "OPENAI_API_KEY", matches = ".+")
+class OpenAiResponsesStreamingChatModelListenerIT extends AbstractStreamingChatModelListenerIT {
+
+    @Override
+    protected StreamingChatModel createModel(ChatModelListener listener) {
+        return OpenAiResponsesStreamingChatModel.builder()
+                .apiKey(System.getenv("OPENAI_API_KEY"))
```

Comment:
```suggestion
                .baseUrl(System.getenv("OPENAI_BASE_URL"))
                .apiKey(System.getenv("OPENAI_API_KEY"))
```
Please add it everywhere

Reply:
My bad, it is failing, pls ignore